### PR TITLE
gst-rtsp-server: Init at 1.12.3

### PR DESCRIPTION
**Pull request by bkchr**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by grahamc** [comment]
@GrahamcOfBorg build gst_all_1.gst-rtsp-server

**Comment by bkchr** [comment]
@grahamc will you merge this? :)

File: pkgs/development/libraries/gstreamer/rtsp-server/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  name = "gst-rtsp-server-1.12.3";
+
+  meta = with stdenv.lib; {
+    description = "Gstreamer RTSP server";
+    homepage    = "https://gstreamer.freedesktop.org";
+    longDescription = ''
+      a library on top of GStreamer for building an RTSP server.
+    '';
+    license     = licenses.lgpl2Plus;
+    platforms   = platforms.linux ++ platforms.darwin;
```
Comment (grahamc) [inline pkgs/development/libraries/gstreamer/rtsp-server/default.nix]: Can you add yourself as a maintainer?

Comment (bkchr) [reply pkgs/development/libraries/gstreamer/rtsp-server/default.nix]: Done

**Pull request merged**